homeassistant/components/webostv/quality_scale.yaml

```diff
@@ -0,0 +1,61 @@
+rules:
+  # Bronze
+  action-setup: todo
```

Comment:
Any reason why the services are not entity services?

Reply:
I will need to look at it deeper, no reason that I can think off